application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1,5 @@
+<?php echo $exhibition['name']; ?>
```

Comment:
__[code_sniffer]__
* File "side_ci.php" should be saved with Unicode (UTF-8) encoding, but it did not successfully pass the W3C test.
* Missing file doc comment
* Any char after the closing PHP tag is prohibited. Please removes newline or spaces after the closing PHP tag.

<!-- Comment by [SideCI](https://sideci.com) -->